helm/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "indexd.fullname" . }}-test-connection"
+  labels:
+    {{- include "indexd.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
```

Comment:
https://helm.sh/docs/topics/chart_tests/

> The job definition must contain the helm test hook annotation: `helm.sh/hook: test`.
>
> ...farther down the same page...
>
> `  annotations:`
> `    "helm.sh/hook": test-success`

I think what you have here is correct, but... :/

Reply:
Haha :/